### ISSUE TITLE
cprng-aes fails to build with crypto-api 0.8

### ISSUE THREAD

**Issue by cwu** [vincenthz/hs-cprng-aes#1]
```
Resolving dependencies...
Configuring cprng-aes-0.2.1...
Preprocessing library cprng-aes-0.2.1...
Building cprng-aes-0.2.1...

Crypto/Random/AESCtr.hs:27:8:
    Could not find module `System.Crypto.Random':
      It is a member of the hidden package `crypto-api-0.7'.
      Perhaps you need to add `crypto-api' to the build-depends in your .cabal file.
      Use -v to see a list of the files searched for.
cabal: Error: some packages failed to install:
```

It seems that they removed System.Crypto.Random in crypto-api-0.8

Forcing to build with crypto-api-0.7 I get 

```
[1 of 1] Compiling Crypto.Random.AESCtr ( Crypto/Random/AESCtr.hs, dist/build/Crypto/Random/AESCtr.o )

Crypto/Random/AESCtr.hs:27:1:
    Warning: Module `System.Crypto.Random' is deprecated:
               Use the 'entropy' package module System.Entropy instead
```

**Comment by vincenthz** [comment]
Thanks ! i've upgraded to newer crypto-api+entropy and released cprng-aes 0.2.2

**Comment by cwu** [comment]
Awesome! thanks for the quick fix
